### DISCUSSION TITLE
Port Ordering

### DISCUSSION THREAD

**Discussion by especially-relative** [emporia-vue-local/esphome#254] [Q&A] [answered]
![image](https://github.com/emporia-vue-local/esphome/assets/10280889/2560909d-2120-4f17-b711-b2cf6f9d5728)

So looking at this Diagram, the top right port is #1, then they are ordered sequentially counterclockerwise,  with the phases C B A on the top... correct?

Then I've got red on the right claim (with red stripes on the breaker) = B, and black on the left clamp (solid black main clamp) = A

I'm using ports 1-6, 15, 16 supposedly

But the readings are all over the place, mostly wrong. I'm showing moderate loads on some of the un-used ports that somewhat make sense?

Does this project use different port numbing than the official docs? Did I miss something?

Thanks!!

**Comment by flaviut** [comment, marked as answer]
Numbering is exactly the same as the documentation/the printing on the unit.

> But the readings are all over the place, mostly wrong

This is pretty much always a phase mismatch. Feel free to remove `*pos` from the filters and see if they're consistently negative.

I also had a bit of trouble with plugs coming slightly unplugged when I put the panel cover on, make sure the clamps plugged in good and tight.

> I'm showing moderate loads on some of the un-used ports that somewhat make sense?

That's expected, just delete the config entries for the unused ports.

**Comment by especially-relative** [comment]
Sorry about that... yes it was indeed a phase mismatch I don't know why I managed to mess it up, I had read the diagrams wrong and shifted the phase of one side (the side with more on it) all off by one...

**Reply by especially-relative** [reply]
Managed to flip red/black wiring harness cables too. The tip to remove the *pos filter was very helpful, thanks!
